CMakeLists.txt

```diff
@@ -16,6 +16,10 @@ if (NOT CMAKE_BUILD_TYPE)
 	set(CMAKE_BUILD_TYPE "Release")
 endif()
 
+if(NOT CMAKE_BUILD_TYPE STREQUAL "Debug")
+	add_definitions(-DNDEBUG)
```

Comment:
I'd rather this go in rpcs3/CMakeLists.txt with the rest of the definitions, line 84-96

Reply:
Moved the conditional to rpcs3/CMakeLists.txt. Does it look good now?

`-DNDEBUG` isn't really Unix-specific.
https://github.com/Kitware/CMake/search?q=-DNDEBUG